typesense/scripts/create-typesense-collection.sh

```diff
@@ -0,0 +1,15 @@
+TYPESENSE_HOST="${TYPESENSE_HOST:-http://localhost:8108}"
+TYPESENSE_API_KEY="${TYPESENSE_API_KEY:-test_api_key}"
+
+curl "${TYPESENSE_HOST}/collections" \
+       -X POST \
+       -H "Content-Type: application/json" \
+       -H "X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}" \
+       -d '{
+         "name": "mm_product_docs",
+         "fields": [
+           {"name": "category", "type": "string" },
+           {"name": "weight", "type": "int32" }
+         ],
+         "default_sorting_field": "weight"
+       }'
```

Comment:
This is how we can prepare the collection to know about how we want to organize certain filterable fields. At the moment, running this script is optional as the scraper will auto generate a collection, but running this is necessary for the field customizations we want to add